lib/cli/scripts/init-aps-env.ts

```diff
@@ -179,24 +178,25 @@ async function hasDefaultTenant(tenantId, tenantName) {
     }
 }
 
-async function createDefaultTenant(tenantName) {
+async function createDefaultTenant(tenantName: string) {
     const tenantPost = {
         active: true,
         maxUsers: 10000,
         name : tenantName
     };
 
     try {
-        const tenant = await alfrescoJsApi.activiti.adminTenantsApi.createTenant(tenantPost);
+        const adminTenantsApi = new AdminTenantsApi(alfrescoJsApi);
+        const tenant = await adminTenantsApi.createTenant(tenantPost);
         logger.info(`APS: Tenant ${tenantName} created with id: ${tenant.id}`);
         return tenant.id;
     } catch (error) {
         logger.info(`APS: not able to create the default tenant: ${JSON.parse(error.message)}` );
     }
 }
 
-async function createUsers(tenandId, user) {
-    logger.info(`Create user ${user.email} on tenant: ${tenandId}`);
+async function createUsers(tenantId: number, user: any) {
```

Comment:
I see in that file multiple places where we have 'any' for user/users. Don't we have any type which we could use for users to avoid any? Like UserRepresentation from js-api or something?

Reply:
ok thank you